public/application.js

```diff
@@ -1,50 +1,72 @@
 $(document).ready(() => {
   play();
+  browserBackButtonClicked();
 });
 
-function play() {
-  $("td").click(e => {
-    e.preventDefault();
-    let request = $.ajax({
+function browserBackButtonClicked(){
+
+  if (window.history && window.history.pushState) {
+
+    window.history.pushState(null, null, '/game');
+    $(window).on('popstate', function() {
+        rewind();
+    });
+  };
+};
+
+function rewind(){
+let request = $.ajax({
         type: "PUT",
-        url: "/game",
-        data: {
-            space: $(e.currentTarget).attr('id'),
-        },
+        url: "/rewind",
         dataType: "json"
     });
-    request.then((response) => {
-      putMarkersDown(response);
-      if (response.over === true) {
-        gameResult(response);
-      };
+    request.then(() => {
+        location.reload();
     });
-  });
+};
+
+function play() {
+$("td").click(e => {
+e.preventDefault();
+let request = $.ajax({
+    type: "PUT",
+    url: "/game",
+    data: {
+        space: $(e.currentTarget).attr('id'),
+    },
+    dataType: "json"
+});
+request.then((response) => {
+    console.log(response);
```

Comment:
this console.log isn't necessary. custom console.log should never be committed to source code.